### DISCUSSION TITLE
Which French Impressionist is famed for his Water Lilies series of paintings?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#82] [Galaxy Trivia] [answered]
**Hint:** CM

**Comment by Tynab** [comment, marked as answer]
The French Impressionist who is famed for his Water Lilies series of paintings is Claude Monet. He created the Water Lilies series in the later years of his life, while he was living at his home in Giverny, France. The series features many paintings of the water lily pond in his garden, with various combinations of colors and light. These paintings are considered some of Monet's greatest works and are now held in collections around the world.

**Reply by ParanoidUser** [reply]
Congrats on earning x2 Galaxy Brain! Keep it up 😉

**Comment by rubikproxy** [comment]
The French Impressionist renowned for his **Water Lilies** series is **Claude Monet**, one of the founders of Impressionism. His masterful series, consisting of around 250 paintings, captures the serene beauty of his water lily pond in Giverny, showcasing his groundbreaking use of light, color, and reflection.
